src/management-system-v2/lib/useParseZodErrors.tsx

```diff
@@ -18,15 +18,18 @@ export default function useParseZodErrors<T extends ZodRawShape>(schema: z.ZodOb
   }
 
   function parseInput(input: unknown) {
+    if (!input) {
+      setErrors({});
+      return null;
+    }
+
```

Comment:
For dx purposes, maybe have reset as a second option, or make a third return value of this hook that is just a `resetErrors()` function. That way only the components that actually need it would use it.

Reply:
Good point, I added resetErrors to the return value